main.cpp

```diff
@@ -0,0 +1,66 @@
+#include "dice.h"
+
+static void parseString(std::string &argument, Dice &obj);
+
+int main(const int argc, const char *argv[]){
+//TODO Add some basic error checking 
+    Dice roller;
+
+    if(argc <= 1){
+        std::cout << "You haven't entered any arguments" << std::endl;
+        std::cout << "The correct usage is: <number of dice>d<sides on dice>" << std::endl;
+        return -1;
+    }
+
+    for(int i = 1; i < argc; i++){
+        //I don't like this can it be cast in the function parameters instead?
+        std::string s (argv[i]);
+        parseString(s, roller);
+    }
+    return 0;
+}
+
+//Removes numbers at beginning and end of string then calls passed function with the arguments
+static void parseString(std::string &argument, Dice &obj){
```

Comment:
I'm not a huge fan of the names "parseString", "argument", or "obj". They could all be more descriptive. :)